src/util/specialKeys.ts

```diff
@@ -0,0 +1,5 @@
+export enum SpecialKeys {
+  ExtensionEnable = '<ExtensionEnable>',
+  ExtensionDisable = '<ExtensionDisable>',
+  TimeoutFinished = '<TimeoutFinished>',
+}
```

Comment:
No final newline - does `gulp forceprettier` do anything?

Reply:
Yes it does! 😅  The problem is that when running build it will only run the prettier on changed files and since some of these files were changed/created before some of the recent changes to prettier settings it did not check them anymore. I'll wait to see if there are more things to review and then I'll push them all together.